kafka/producer.go

```diff
@@ -0,0 +1,59 @@
+// Package kafka provides producer and consumer to work with kafka topics
+package kafka
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	"github.com/lvl484/positioning-filter/position"
+)
+
+type Producer interface {
+	Produce(position.Position) error
+	Close() error
+}
+
+type producer struct {
+	KafkaProducer sarama.SyncProducer
+	Config        *Config
+}
+
+func NewProducer(config *Config) (Producer, error) {
+	addr := []string{fmt.Sprintf("%v:%v", config.Host, config.Port)}
+	saramaProducer, err := sarama.NewSyncProducer(addr, nil)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &producer{
+		KafkaProducer: saramaProducer,
+		Config:        config,
+	}, nil
+}
+
+// Produce message with filtered position to kafka topic "filtered-positions"
+func (p *producer) Produce(pos position.Position) error {
```

Comment:
It would be great to declare Message type

Reply:
fixed